internal/fluentbit/config/builder/config_builder.go

```diff
@@ -15,8 +15,13 @@ type PipelineDefaults struct {
 	FsBufferLimit     string
 }
 
+type BuilderConfig struct {
```

Comment:
Here it was on purpose. Since we thought we reffer to the configuration of the FluentBit builder, not the configuration of the config_builder itself. But if you still think using Config is better, I can change it, no problem.

Reply:
Good point, let's keep it.